adobe/spectrum-web-components
# fix(combobox): add missing React TS typings

## Description

Adds missing components to `tsconfig-react-wrapper.json`.  Should this be a step in the documentation as well?

## Related issue(s)



- https://github.com/adobe/spectrum-web-components/issues/4418

## Motivation and context


Consumers complained for the lack of TS definitions for Combobox when using `@swc-react` in a project with Typescript.
I added other missing component definitions as well.

## How has this been tested?




    1. Run `yarn build:react`
    2. Open the `/react` folder
    3. Open the packages modified in this PR
    4. Observe `.d.ts` files exist.

## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
